yt/geometry/selection_routines.pyx

```diff
@@ -881,8 +890,16 @@ cdef class RegionSelector(SelectorObject):
     @cython.cdivision(True)
     cdef int select_bbox(self, np.float64_t left_edge[3],
                                np.float64_t right_edge[3]) nogil:
+        return self.select_bbox_ndims(left_edge, right_edge, 3)
```

Comment:
Could/should we make this in the base class, and then have the subclasses implement `select_bbox_ndims` instead?